src/recommendations/product-recs.md

```diff
@@ -0,0 +1,53 @@
+---
+group: product-recommendations
+title: Product Recommendations
+---
+
+Product recommendations are a powerful marketing tool you can use to increase conversions, boost revenue, and stimulate shopper engagement. Recommendations are surfaced on the storefront in the form of units such as “Customers who viewed this product also viewed”. Because these suggestions are backed by a deep analysis of aggregated visitor data, they result in highly engaging, relevant, and personalized experiences for the shopper. Magento does not collect personally identifiable information.
+
+Using recommendations on your storefront requires:
+
+-  **Behavioral data** - Computes the product affinities used as the basis for recommendations
+
+-  **Catalog data** - Displays product metadata (name, price, availability, and so on) in recommendations
+
+-  **Admin UI** - Allows you to configure recommendations on your storefront
+
+## Install Product Recommendations {#install}
```

Comment:
This appears to be duplicate content with the `install.md` file. If you're required to maintain this information in two places, use an include file.

Reply:
the install.md will be going away.